src/core/public/chrome/nav_controls/nav_controls_service.ts

```diff
@@ -130,6 +140,11 @@ export class NavControlsService {
           map((controls) => sortBy([...controls.values()], 'order')),
           takeUntil(this.stop$)
         ),
+      getLeftBottom$: () =>
+        navControlsLeftBottom$.pipe(
+          map((controls) => sortBy([...controls.values()], 'order')),
```

Comment:
okay, so registration uses set to dedup, then when get, there are ordered?

Reply:
Yes.